src/core/analytics.js

```diff
@@ -1133,6 +1146,10 @@ class Analytics {
       transformToRudderNames(this.loadOnlyIntegrations);
     }
 
+    if (options && options.useIntegrationsInEvents) {
```

Comment:
rename this to useGlobalIntegrationsConfigInEvents or something so it is more explanatory.

current name useIntegrationsInEvents could be confusing as it will still sent them in events level if defined in the apiOptions